api-extractor/testInputs/example2/folder/MyDocumentedClass.ts

```diff
@@ -7,7 +7,7 @@ export enum TestMissingCommentStar {
 }
 
 /**
- * {@inheritdoc es6-collections:ForEachable}
+ * {@inheritdoc @types/es6-collections:ForEachable}
```

Comment:
This is what the inheritdoc must look like for @types packages now. Previously, when had 'es6-collections:ForEachable', the symbol lookup returns a path containing the scope name '@microsoft', and so the system was looking in the cache for a key '@microsoft/es6-collections' since we weren't explicit providing a scope name.